src/components/createdCharPage.js

```diff
@@ -1,13 +1,16 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { Navigate } from 'react-router';
 
 import { Header, Footer } from './header-footer'
 
-export default function CreatedCharPage(props) {
-   const { user } = props
+//State Management - Context API
+import { UserContext } from '../contextAPI';
+
+export default function CreatedCharPage() {
+   const user = useContext(UserContext)
```

Comment:
Nit: might want to change this to userContext so when you destructure it doesn't look as odd as user.user.first_name